From: Artemy Kovalyov <artemyko@mellanox.com>
Date: Wed, 31 Aug 2016 05:17:54 +0000
Subject: net/mlx5: Update struct mlx5_ifc_xrqc_bits
Patch-mainline: v4.10-rc1
Git-commit: 5579e1519bad43b874922dbe87c74fdcbd97a7db
References: bsc#1015342 FATE#321688 bsc#1015343 FATE#321689

Update struct mlx5_ifc_xrqc_bits according to last specification

Signed-off-by: Artemy Kovalyov <artemyko@mellanox.com>
Signed-off-by: Leon Romanovsky <leon@kernel.org>
Acked-by: Benjamin Poirier <bpoirier@suse.com>
---
 include/linux/mlx5/mlx5_ifc.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/include/linux/mlx5/mlx5_ifc.h
+++ b/include/linux/mlx5/mlx5_ifc.h
@@ -2844,7 +2844,7 @@ struct mlx5_ifc_xrqc_bits {
 
 	struct mlx5_ifc_tag_matching_topology_context_bits tag_matching_topology_context;
 
-	u8         reserved_at_180[0x200];
+	u8         reserved_at_180[0x880];
 
 	struct mlx5_ifc_wq_bits wq;
 };
